interface-definitions/include/firewall/default-action-bridge.xml.i

```diff
@@ -0,0 +1,34 @@
+<!-- include start from firewall/default-action-bridge.xml.i -->
+<leafNode name="default-action">
+  <properties>
+    <help>Default-action for rule-set</help>
```

Comment:
Why a hyphen? "Default action" is not a compound word, it's just an adjective with a noun.